src/Workspaces/Core/Portable/CodeGeneration/Symbols/CodeGenerationAbstractNamedTypeSymbol.cs

```diff
@@ -122,6 +122,6 @@ public override string MetadataName
 
         public bool IsSerializable => false;
 
-        public bool IsFile => Modifiers.IsFile;
+        public bool IsFileLocal => Modifiers.IsFile;
```

Comment:
>IsFileLocal [](http://example.com/codeflow?start=20&length=11)

I'm having a slight hesitation, whether this should be `IsFileLocalType`, like `IsUnmanagedType`, `IsRefLikeType` or `IsGenericType` above...

Reply:
I don't feel strongly about this here. It's an internal API so don't think there's a rush to make it happen.

Reply:
@RikkiGibson But that applies to the public `INamedTypeSymbol.IsFileType` property as well?

Reply:
Oops, I could have placed the comment in a better location ;-)

> we might end up introducing API like ISymbol.IsFileLocal and then the INamedTypeSymbol.IsFileLocal would shadow it.

Makes sense. Thanks